docs/vsc.md

```diff
@@ -0,0 +1,14 @@
+# Vcs Connector
+
+Vcs connector is a feature that lets you use different vsc system with one commands interface.
+
+## How to run a vsc
+
+Use the following commands for tasks:
+
+- To run a task: `spryker-sdk sdk:vcs:clone`
+  - After executing the command, you will be prompted to fill in the repository and select a specific vsc.
+
+## Where to find cloned project
+
+All project will be cloned in temporary `var` directory.
```

Comment:
looks like we missed some requirements.
Should be cloned into the `var/tmp`, because `var` is not temporary directory and we have some folders in it. Please, update the command.